README.md

```diff
@@ -92,22 +92,42 @@ To use the Material theme, import the correspondent file from the `theme/materia
   `vaadin-element.html`
 
 
-## Running demos and tests in browser
+## Running demos and tests in a browser
+Ensure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
 
-1. Fork the `vaadin-element` repository and clone it locally.
+Clone the `vaadin-select` repository.
```

Comment:
please update to `vaadin-element` here and below.

Reply:
OK